src/utils/wrappers.ts

```diff
@@ -386,21 +424,27 @@ export function handleWrappers(ast: Node) {
         wrapper.callee.body.body[3].expression.right.type === 'ObjectExpression' &&
         (wrapper.callee.body.body[4].type === 'VariableDeclaration' &&
           wrapper.callee.body.body[4].declarations.length === 1 &&
+          wrapper.callee.body.body[4].declarations[0].init &&
           wrapper.callee.body.body[4].declarations[0].init.type === 'CallExpression' &&
           wrapper.callee.body.body[4].declarations[0].init.callee.type === 'Identifier' &&
           wrapper.callee.body.body[4].declarations[0].init.callee.name === 'require' ||
           wrapper.callee.body.body[5].type === 'VariableDeclaration' &&
           wrapper.callee.body.body[5].declarations.length === 1 &&
+          wrapper.callee.body.body[5].declarations[0].init &&
           wrapper.callee.body.body[5].declarations[0].init.type === 'CallExpression' &&
           wrapper.callee.body.body[5].declarations[0].init.callee.type === 'Identifier' &&
           wrapper.callee.body.body[5].declarations[0].init.callee.name === 'require')) {
       const externalMap = new Map<number, any>();
-      const moduleObj = wrapper.callee.params.length ? wrapper.arguments[0] : wrapper.callee.body.body[3].expression.right;
-      let modules: [number, any][];
+      const statement = wrapper.callee.body.body[3];
+      if (statement.type !== 'ExpressionStatement' || statement.expression.type !== 'AssignmentExpression' || statement.expression.right.type !== 'ObjectExpression') {
+        throw new Error('Expected ExpressionStatement');
+      }
+      const moduleObj = wrapper.callee.params.length ? wrapper.arguments[0] : statement.expression.right;
+      let modules: [number, typeof moduleObj][];
       if (moduleObj.type === 'ArrayExpression')
-        modules = moduleObj.elements.map((el: any, i: number) => [i, el]);
+        modules = moduleObj.elements.map((el, i) => [i, el]);
       else
-        modules = moduleObj.properties.map((prop: any) => [prop.key.value, prop.value]);
+        modules = 'properties' in moduleObj ? moduleObj.properties.map(prop => [prop.key.value, prop.value]) : [];
```

Comment:
@guybedford I added the check for existence of `properties` but the remaining problem here is that `prop.key` might not exist. Should I Filter out all the items where `prop.key` doesnt exist?